molecule/default/playbook.yml

```diff
@@ -31,5 +31,66 @@
       args:
         port: 22
         state: started
-        delay: 300
-        timeout: 420
+        delay: 600
+        timeout: 1200
+
+- name: Reboot infra3 hosts
+  hosts: infra3
```

Comment:
You can collapse the individual tasks for the `Reboot infra* hosts` tasks by using the `order` directive on the host group to reboot them in reverse order:

```
- name: Reboot os-infra_hosts
  hosts: os-infra_hosts
  order: reverse_inventory
  tasks:
    - shell: sleep 2 && shutdown -r now "Ansible reboot {{ inventory_hostname }} host"
      ...
```